src/store/reducers/index.ts

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from "redux";
 import ethereum from "./ethereum";
 import tezos from "./tezos";
+import notifications from './notifications';
 
 export default combineReducers({
   ethereum,
+  notifications,
```

Comment:
Ja bym dała notifications jako pierwszy, żeby nie mieszał się z blockchainami :) ale to taka sugestia

Reply:
Przyjęłam strategię sortowania alfabetycznego. Potem w devtoolsach w storze to i tak będzie alfabetycznie.